make/data/macosxsigning/default-debug.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>com.apple.security.cs.allow-jit</key>
+    <true/>
+    <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
+    <true/>
+    <key>com.apple.security.cs.disable-library-validation</key>
+    <true/>
+    <key>com.apple.security.cs.allow-dyld-environment-variables</key>
+    <true/>
+    <key>com.apple.security.cs.debugger</key>
+    <true/>
+    <key>com.apple.security.get-task-allow</key>
```

Comment:
If instead of creating this new plist file you instead just added this property to default.plist, would that cause notarization to fail, or would notarizing simply strip this entitlement, which is what you would want.

Reply:
Yes, that would cause notarization to fail. Note that notarization does not change the binaries. We just send them in to be scanned by Apple, and they save some kind of fingerprint in a database. (There is a post notarization step called stapling that does modify the binaries, but all it does is attaching a signature from the notarization process to make it possible to verify notarization offline.)

I could have opted to generate the plist files using PlistBuddy instead of keeping 4 static files around, but decided against it to keep the build logic simpler. If we ever need any more variants, I will probably rethink this.